packages/babel-plugin-transform-destructuring/src/index.ts

```diff
@@ -50,20 +50,16 @@ export default declare((api, options: Options) => {
         if (!declaration.isVariableDeclaration()) return;
         if (!variableDeclarationHasPattern(declaration.node)) return;
 
-        const specifiers = [];
-
-        for (const name of Object.keys(path.getOuterBindingIdentifiers())) {
-          specifiers.push(
-            t.exportSpecifier(t.identifier(name), t.identifier(name)),
-          );
-        }
-
         // Split the declaration and export list into two declarations so that the variable
         // declaration can be split up later without needing to worry about not being a
         // top-level statement.
-        path.replaceWith(declaration.node);
-        path.insertAfter(t.exportNamedDeclaration(null, specifiers));
-        path.scope.crawl();
```

Comment:
The `crawl` is also removed since `splitExportDeclaration` will take care of scope info updates.